client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -68,6 +76,43 @@ class DebugTab extends Component {
 							</FormToggle>
 						</FormFieldset>
 						<div className="wp-super-cache__debug-fieldsets">
+							{ wp_cache_debug_log &&
+								<table>
+									<tr>
+										<td>
+											{ wp_super_cache_debug
+												? translate( 'Currently logging to:' )
+												: translate( 'Last logged to:' )
+											}
+										</td>
+										<td>
+											<ExternalLink
+												href={ cache_path_url + wp_cache_debug_log }
+												target="_blank">
+												{ cache_path + wp_cache_debug_log }
+											</ExternalLink>
+										</td>
+										<td rowSpan="2">
+											<Button
+												compact
+												disabled={ isRequesting || isSaving }
+												onClick={ this.deleteLog }
+												scary
+												value="1">
```

Comment:
Ah, good point. This might be a leftover from an earlier approach of ensuring we'd set `wpsc_delete_log` to `1` using `POST`, but it's probably [not needed](https://github.com/Automattic/wp-calypso/pull/16282/files#diff-82589d7f87af9eb87adbd3592a7b7ac4R33) any longer.